src/features/usage-consolidation-accordion/usage-consolidation-accordion.js

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+
+import { useStripes } from '@folio/stripes/core';
+import {
+  Accordion,
+  Headline,
+} from '@folio/stripes/components';
+
+import Toaster from '../../components/toaster';
+import { getUsageConsolidation as getUsageConsolidationAction } from '../../redux/actions';
+import { selectPropFromData } from '../../redux/selectors';
+
+const propTypes = {
+  getUsageConsolidation: PropTypes.func.isRequired,
+  headerProps: PropTypes.object,
+  id: PropTypes.string.isRequired,
+  isOpen: PropTypes.bool,
+  onToggle: PropTypes.func.isRequired,
+  section: PropTypes.node.isRequired,
+  usageConsolidation: PropTypes.shape({
+    data: PropTypes.object,
+    errors: PropTypes.array,
+    isFailed: PropTypes.bool.isRequired,
```

Comment:
I see you update usage consolidation reducer, please update usageConsolidation shape with `isLoaded` and `isLoading`. Also I think it will be great to create shape for this prop and use it in other files.

P.S. `data` and `errors` are required props too.